wiki/Tournaments/The_Roundtable/16_player_battle_royale/en.md

```diff
@@ -0,0 +1,64 @@
+---
+tags:
+  - The Round table
+---
+
+# The Roundtable 16 player battle royale
+
+![The Roundtable banner](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+[**The Roundtable**](/wiki/Tournaments/The_Roundtable)'s **16 player battle royale** takes place the first day of the event.
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge |
+
+## Participants
+
+| User |
+| :-- |
+| ![][flag_AU] [sakamata1](https://osu.ppy.sh/users/7562902) |
```

Comment:
```suggestion
| ::{ flag=AU }:: [sakamata1](https://osu.ppy.sh/users/7562902) |
```
should write country flags like this now, and you don't need the references if done this way